docs/advanced_usage/how_to_change_logger_class.rst

```diff
@@ -0,0 +1,40 @@
+.. _change_logger_class:
+
+
+How to change the class of the logger
```

Comment:
Why would it be needed? Which use-case did we accounted?

Reply:
The use case is to use kiso testing with fluentd to use structured logger, for this a logger using : https://github.com/fluent/fluent-logger-python#python-logginghandler-interface can be used.
So the log of the test could be send with the handler of the new Logger Class just by calling `pykiso --logger `